src/amo/pages/LanguageTools/index.js

```diff
@@ -175,6 +158,8 @@ export class LanguageToolsBase extends React.Component<Props> {
           />
         </Helmet>
 
+        <HeadLinks to="/language-tools/" />
```

Comment:
I thought about doing something similar but the `prefixMiddleware` logic is super obscure (with config calls everywhere and so on). Having hardcoded values is less nice but it is simpler and likely faster. I could probably create exported variables for all the static paths, that'd help a bit. A more lame argument would be that we never changed URLs yet, that doing this would likely affect SEO and also one (like QA) would notice a breaking change.

> Let me know what you think.

To sum up, I don't know 😶